views/lyrics.tmpl

```diff
@@ -11,7 +11,7 @@
 		{{template "navbar"}}
 		<div id="container">
 			<div id="metadata">
-				<img src="{{extractURL .Image}}"/>
+				<a href="{{.LinkToAlbum}}"><img src="{{extractURL .Image}}"/></a>
```

Comment:
The image width was increased by this, maybe we should add something like `max-width: 200px`